cmd/pluginator/internal/krmfunction/converter.go

```diff
@@ -147,6 +147,10 @@ func (c *Converter) readDiskFile(path string) (string, error) {
 func (c *Converter) mkDstDir() error {
 	p := c.outputDir
 	f, err := os.Open(p)
+	if f != nil {
+		f.Close()
+    	}
+	
```

Comment:
It looks like the only purpose for this `os.Open()` call is to check whether the folder already exists. Since this is the case, I believe it would be better to use `os.Stat()` here instead of `os.Open()`.